lib/wpMandrill.class.php

```diff
@@ -309,8 +309,7 @@ static function showOptionsPage() {
                 <?php
                 $rss        = fetch_feed('http://blog.mandrill.com/feeds/all.atom.xml');
                 $maxitems   = 0;
-                if (!is_wp_error( $rss ) ) {
-                    $maxitems = $rss->get_item_quantity(5);
+                if (!is_wp_error( $rss ) && $maxitems = @$rss->get_item_quantity(5)) {
```

Comment:
@bizmate  Is any of this code needed at all? It seems on first look that all of the code of the outer div (starting on 308) can be removed and this check doesn't even need to happen.

Reply:
Depends in if we want to keep rendering the Mandrill news if and when the core library depending on simplepie gets fixed.

Indeed it is a decision between this PR and yours, either way is fine for us but I have taken a fall back approach in this one so that if and when the core library gets fixed the block will render again. You can close this PR and merge yours instead if you want to take the approach of actually removing the whole news rendering block.

One thing i missed is the removal of the translation for the sentence used in this code. 
https://github.com/Miller-Media/send-emails-with-mandrill/blob/741b545505b127612e5f34c4e0c9e0fcad17ffe0/lang/wpmandrill.po#L258-L264

I will commit the change if you would like to keep this PR

Reply:
Let's remove the code and then we can keep a to-do on this open as an 'issue' in GitHub to track it. As best practice, we should remove any code that isn't being used from the codeset but can revisit at a later date if SimplePie gets fixed (it doesn't seem to be high priority for them). Happy to merge this branch in if you want to make that commit. Thanks!